components/service/glean/scripts/sdk_generator.gradle

```diff
@@ -273,6 +275,54 @@ ext.gleanGenerateMetricsAPI = {
     }
 }
 
+// Generate the Metrics docs, if requested.
+if (project.ext.has("gleanGenerateMarkdownDocs")) {
+    task gleanGenerateMetricsDocs(type: Exec) {
+        description = "Generate the Markdown docs for the collected metrics"
+
+        outputs.dir "$projectDir/docs"
+        workingDir rootDir
+
+        // Note that the command line is OS dependant: on linux/mac is Miniconda3/bin/python.
+        if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+            commandLine new File(GLEAN_MINICONDA_DIR, "python")
+        } else {
+            commandLine new File(GLEAN_MINICONDA_DIR, "bin/python")
+        }
```

Comment:
(Maybe) pull this out into a function to reduce code duplication with the Kotlin generator...

Reply:
Done!